### PR TITLE
Virtualize columns

### PR DESCRIPTION
**Pull request by sklinov**

### CODE REVIEW

File: src/GridTable.js [hunk 1]
```diff
@@ -117,13 +128,13 @@ class GridTable extends React.PureComponent {
           frozenData={frozenData}
           width={width}
           height={Math.max(height - headerHeight - frozenRowsHeight, 0)}
-          rowHeight={estimatedRowHeight ? getRowHeight : rowHeight}
+          rowHeight={(estimatedRowHeight || columnWidths) ? (getRowHeight ? getRowHeight : () => rowHeight)  : rowHeight}
```
Comment (dagadbm) [inline src/GridTable.js:131]: why does it need to be a function? it sems they all return rowHeight in the end?

File: types/index.d.ts [hunk 2]
```diff
@@ -426,6 +426,14 @@ declare module '@datafoldcom/react-base-table' {
      * Number of rows to render above/below the visible bounds of the list
      */
     overscanRowCount?: number;
+    /**
+    * Number of rows to render above/below the visible bounds of the list
```
Comment (dagadbm) [inline types/index.d.ts]: rows or columns?

**Review by dagadbm — APPROVED** [review]
left some comments, approved to not block.

File: src/GridTable.js (hunk 1 above)
Comment (sklinov) [reply src/GridTable.js:131]: The signature is different in different modes.
Check out like 115 where selection is made between VariableSizeGrid and FixedSizeGrid

File: types/index.d.ts (hunk 2 above)
Comment (sklinov) [reply types/index.d.ts]: Right, that was wrong copy-pasting. Will fix now

**Pull request merged**